### PR TITLE
Added System.Formatters.Nrbf to Version.Details.xml

### PR DESCRIPTION
**Pull request by dipeshmsft**
###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/dotnet/wpf/pull/9517)

### CODE REVIEW

File: eng/Version.Details.xml [hunk 1]
```diff
@@ -95,6 +95,10 @@
       <Uri>https://github.com/dotnet/runtime</Uri>
       <Sha>7f2b222f92d323b7ca522a06c7116f1ec25d12ca</Sha>
     </Dependency>
+    <Dependency Name="System.Formats.Nrbf" Version="9.0.0-preview.7.24401.8" CoherentParentDependency="Microsoft.Private.Winforms">
```
Comment (anjali-wpf) [inline eng/Version.Details.xml:98]: Are we going to add reference for this in versions.props as well?

Comment (pchaurasia14) [reply eng/Version.Details.xml:98]: Version.props already has the reference to this package. We missed adding it here.

Comment (pchaurasia14) [inline eng/Version.Details.xml:98]: @dipeshmsft - Check the version number. Should be rc1 version.

Comment (ericstj) [reply eng/Version.Details.xml:98]: Correct, this needs to match the other entries here.  It should also match what's in Versions.props.

**Review by pchaurasia14 — APPROVED** [review]

**Review by harshit7962 — APPROVED** [review]

**Pull request merged**